payjoin/src/receive/mod.rs

```diff
@@ -481,6 +481,16 @@ impl ProvisionalProposal {
         self.payjoin_psbt.unsigned_tx.output[self.owned_vouts[0]].script_pubkey = script;
     }
 
+    /// payjoin request value in satoshis
```

Comment:
This is useful in lightning context in order to get a scheduled channel based on the payjoin amount received
exmple: https://github.com/lightningdevkit/ldk-node/pull/257/files#diff-d07cd679b2d6743b550aec08c91b33e92bdd40a9556385925c987e9d82b222c5R89